frontend/components/common/Login.tsx

```diff
@@ -1,28 +1,50 @@
-import React from "react";
+import React , {useState, useEffect}from "react";
 import { ConnectButton } from "web3uikit";
 
 function Login() {
+  const [isSmallScreen, setIsSmallScreen] = useState(false);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setIsSmallScreen(window.innerWidth <= 600);
+    };
+
+    // Add event listener to track window resize
+    window.addEventListener("resize", handleResize);
+
+    // Initial check for small screen
+    handleResize();
+
+    // Clean up the event listener on unmount
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
+
+
   return (
     <div>
       <div
         style={{
           minHeight: "90vh",
           display: "flex",
-          justifyContent: "center",
-          flexDirection: "column",
-          textAlign: "center",
-          alignContent: "center",
+          justifyContent: "center",     
           alignItems: "center",
           gap: "20px",
+          flexDirection: isSmallScreen ? "column" : "row", 
         }}
       >
         <img
           src="/images/welcome.svg"
           alt="no data"
-          width={"100vh"}
-          height={"100vh"}
+          width={"300vh"}
+          height={"300vh"}

```

Comment:
why have you done this ? 

i am assuming the img was too small ?
@BhartiNagpure